model/src/ww3_ounf.F90

```diff
@@ -1358,16 +1348,6 @@ SUBROUTINE W3EXNC ( NX, NY, IX1, IXN, IY1, IYN, NSEA,             &
               IF ( FLAGUNR ) CALL W3THRTN(NSEA, THP0, AnglD, .FALSE.)
 #endif
               CALL S2GRID(THP0, X1, .TRUE.)
-!              IF( SMCGRD ) THEN
```

Comment:
@ukmo-ccbunney are you okay with this cleanup?  It looks related to SMCGRID so just wanted to double check.